modules/cnf-managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus.adoc

```diff
@@ -0,0 +1,58 @@
+// CNF-802 Infrastructure-provided interrupt processing for guaranteed pod CPUs
+// Module included in the following assemblies:
+//
+// *cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+[id="managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus_{context}"]
+= Managing device interrupt processing for guaranteed pod isolated CPUs
+
+The Performance Addon Operator manages host CPUs by dividing them into reserved CPUs for cluster housekeeping duties, and isolated CPUs for workloads. CPUs that are used for real-time workloads are set as isolated. Being able to configure device interrupt processing means that administrators can balance the number of reserved CPUs against the number of guaranteed pod CPUs for low-latency processing. In the performance profile, `GloballyDisableIrqLoadBalancing` is used to manage whether device interrupts are processed or not.
```

Comment:
for cluster housekeeping duties - > for cluster and Operating System housekeeping duties

Reply:
CPUs that are used for real-time workloads -> CPUs that are used for low-latency workloads

Reply:
"Being able to configure device interrupt processing means that administrators can balance the number of reserved CPUs against the number of guaranteed pod CPUs for low-latency processing." -> The sentence is not clear and not entirely accurate.

Being able to load balance device interrupts means the system can service interrupts on different CPUs and is able to move the device interrupts between CPUs in order to avoid one or more CPUs overloading leading to performance degradation.

In our context that means the system can load-balance the device interrupts between ALL CPUs Isolated or Reserved, as long as there is no guaranteed pod running on them.

The "dynamic" part in our case is the ability to "ban" guaranteed pods CPUs from servicing device interrupts when the mentioned annotations are set for the pod.